public/paths/hubs/roles/role.yml

```diff
@@ -0,0 +1,120 @@
+get:
+  operationId: "getRole"
+  tags:
+    - Role
+  parameters:
+    - name: roleId
+      description: The ID of the Role.
+      in: path
+      required: true
+      schema:
+        type: string
+  summary: Fetch Role
+  description: Requries the `hubs-roles-manage` capability.
+  responses:
+    200:
+      description: Returns an Role resource.
```

Comment:
'an Role resource' -> 'a Role'. Drop resource. Again, look at the pattern on my new branch.